clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -9979,7 +9971,7 @@ static void ProcessDeclAttribute(Sema &S, Scope *scope, Decl *D,
     handleSYCLIntelNoGlobalWorkOffsetAttr(S, D, AL);
     break;
   case ParsedAttr::AT_SYCLIntelUseStallEnableClusters:
-    handleUseStallEnableClustersAttr(S, D, AL);
+    handleSYCLIntelUseStallEnableClustersAttr(S, D, AL);
```

Comment:
Why do we need to do this at all instead of setting `SimpleHandler = 1` in Attr.td?

Reply:
Thank you @AaronBallman for the review. We still need this because in downstream, we have OpenCL/HLS attribute (GNU spelling) that uses same source codes and we do extra checking's like (Check GNU vs SYCL spellings and Deprecated GNU spelling) there.

Reply:
I'm not opposed, but this does make for one of those Interesting Situations given that we also are supposed to try to upstream this sort of stuff. :-D

Reply:
I agree with you, @AaronBallman. I will work on refactoring downstream codes to resolve this problem.